.eslintrc.json

```
@@ -0,0 +1,15 @@
+{
+	"env": {
+		"es6": true,
+		"node": true
+	},
+	"parserOptions": {
+		"sourceType": "module",
+		"ecmaVersion": 2019
+	},
+	"plugins": ["prettier"],
+	"extends": ["plugin:prettier/recommended"],
+	"rules": {
+		"prefer-expect-assertions": 0
```

Active prettier on eslint
```suggestion
		"prefer-expect-assertions": 0,
		"prettier/prettier": "error"
```